app/controllers/static_pages_controller.rb

```diff
@@ -17,6 +17,15 @@ def index
       @service = StaticPageService::Index.new(current_user:)
 
       @events = @service.events
+
+      featured_event_ids = %w[org_MpJurQ org_Y0zun7 org_Y1ZuDz org_DyuReR org_Jounxy org_0zuXDP org_1Zu4Jr org_5Gu7Lo org_E1uGdn org_G3uq7b]
+
+      @featured_events = featured_event_ids.map do |id|
+        Event.find_by_public_id(id)
+      end.select do |event|
+        event&.is_public? && event.is_indexable?
+      end.shuffle.sample(6)
```

Comment:
```suggestion
      end.sample(6)
```
I think when using `sample`, there's no need to also `shuffle`.